sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -268,14 +266,14 @@ def load_attr(state, arg):
   name = state.get_name(arg)
   if isinstance(o, Const) and hasattr(o.value, name):
     state.stack.append(Const(getattr(o.value, name)))
-  elif (inspect.isclass(o) and
-        isinstance(getattr(o, name, None),
-                   (types.MethodType, types.FunctionType))):
+  elif (inspect.isclass(o)
```

Comment:
Not sure I like this change: `and` should be at the end of this line.

Reply:
I believe the knob
SPLIT_BEFORE_LOGICAL_OPERATOR = false
would change this.

Reply:
That's right, it worked